mirrord/cli/src/error.rs

```diff
@@ -236,6 +236,13 @@ pub(crate) enum CliError {
     #[error("Failed to prepare mirrord operator client certificate: {0}")]
     #[diagnostic(help("{GENERAL_BUG}"))]
     OperatorClientCertError(String),
+
+    #[error("mirrord operator was not found in the cluster.")]
+    #[diagnostic(help(
+        "Used command requires a mirrord operator or operator usage was explicitly enabled in the configuration file.
+        Install the operator using the `mirrord operator setup` command or the Helm chart at https://github.com/metalbear-co/charts/tree/main/mirrord-operator.{GENERAL_HELP}"
```

Comment:
```suggestion
        You can install the operator with `mirrord operator setup` or by using the Helm chart at https://github.com/metalbear-co/charts/tree/main/mirrord-operator.{GENERAL_HELP}"
```

Maybe? Should we also mention you can get an operator trial free + no credit card?

Reply:
Hmm, not sure. @aviramha ?
Context - this error is shown when we need the operator and we not for sure that it is not installed

Reply:
@eyalb181 maybe?  : D

Reply:
This needs to address two cases: 1. user registered for mfT but hasn't installed the Operator, 2. user hasn't even registered for mfT.
I think it should just be "Read more here" with this link: https://mirrord.dev/docs/overview/quick-start/#operator